Code/Project/Project/FileWritePage.xaml.cs

```diff
@@ -0,0 +1,116 @@
+            DialogResult result = dialog.ShowDialog();
+
+            if(result == DialogResult.OK)
+            {
+                string path = dialog.SelectedPath + "\\SCRUM_Session_" + DateTime.Now.ToString("dd/MM/yyyy") + ".txt";
+                StreamWriter writer = new StreamWriter(path);
+
+                writer.WriteLine(DateTime.Now.ToString());
+                writer.WriteLine();
+                writer.WriteLine("Project: " + Global.ProjectName);
+                writer.WriteLine("SCRUM master: " + Global.ScrumMasterName);
+                writer.WriteLine("==================================");
+
+                foreach (Task task in Global.Tasks)
+                {
+                    writer.WriteLine(task.Name);
+                    writer.WriteLine(task.Description);
+                    writer.WriteLine("Story points: " + task.Rating);
+                    writer.WriteLine("==================================");
+                }
+
+                writer.WriteLine("Generated " + Global.Tasks.Count + " tasks");
+                writer.Close();
+            }
+        }
+
+        private void JsonExport(object sender, RoutedEventArgs e)
+        {
+
+            var dialog = new FolderBrowserDialog();
+            DialogResult result = dialog.ShowDialog();
+
+            if (result == DialogResult.OK)
+            {
+                string path = dialog.SelectedPath + "\\SCRUM_Session_" + DateTime.Now.ToString("dd/MM/yyyy") + ".json";
+                StreamWriter writer = new StreamWriter(path);
+
+                JsonOutput json = new JsonOutput
+                {
+                    ProjectName = Global.ProjectName,
+                    ScrumMasterName = Global.ScrumMasterName,
+                    CurrentDate = DateTime.Now.ToString(),
+                    TaskCount = Global.Tasks.Count,
+                    Tasks = Global.Tasks
+                };
+
+                writer.WriteLine(JsonConvert.SerializeObject(json, Formatting.Indented));
+
+                writer.Close();
+            }
+        }
+
+        private void ExitPoker(object sender, RoutedEventArgs e)
+        {
+            System.Windows.Application.Current.Shutdown();
+        }
+
+    }
+
+    internal class JsonOutput
+    {
+        private string projectName = "";
+        private string scrumMasterName = "";
+        private string currentDate = "";
+        private int taskCount;
+        private List<Task> tasks = new List<Task>();
+
+        public string ProjectName { get { return projectName; } set { projectName = value; } }
+        public string ScrumMasterName { get { return scrumMasterName; } set {  scrumMasterName = value; } }
+        public string CurrentDate { get {  return currentDate; } set {  currentDate = value; } }
+        public int TaskCount { get {  return taskCount; } set {  taskCount = value; } }
+        public List<Task> Tasks { get {  return tasks; } set { tasks = value; } }

```

Comment:
You dont need the private versions of the vars. "pulbic Type Var" would be enough.

Reply:
sure